src/screens/BiometricsPrompt/BiometricsPrompt.js

```diff
@@ -128,12 +130,16 @@ class BiometricsPrompt extends React.Component<Props> {
     const biometryType = navigation.getParam('biometryType');
     const biometryTypeTitle = getBiometryType(biometryType);
     const imageSource = getBiometryImage(biometryType);
+    const colors = getThemeColors(this.props.theme);
     return (
       <ContainerWithHeader headerProps={{ centerItems: [{ title: 'Make crypto easy' }] }}>
         <ContentWrapper contentContainerStyle={{ paddingVertical: 20, paddingHorizontal: 30, flexGrow: 1 }}>
           <HeaderText>{`Would you like to use\n${biometryTypeTitle} with your\nwallet?`}</HeaderText>
           <ContentInnerWrapper>
-            <TouchIdImage source={imageSource} />
+            <TouchIdImage
+              source={imageSource}
+              style={{ tintColor: colors.positive }}
```

Comment:
can we add this style in a styled component? like this? note I haven't tested if that works :p 
```
const TouchIdImage = styled(CachedImage)`
  width: 164px;
  height: 164px;
  tint-color: ${themedColors.positive};
`;
```